### ISSUE TITLE
NO_PROXY config for on premise locations

### ISSUE THREAD

**Issue by uniquecorn09** [ibm-cloud-docs/satellite#43]
The doc is confusing regarding setting up the mirror location ($REDHAT_PACKAGE_MIRROR_LOCATION)  for an ibmcloud satellite location on premise (RHEL 8.7).  There is no ENV $REDHAT_PACKAGE_MIRROR_LOCATION defined. The only repo config we have is /etc/yum.repos.d/redhat.repo, with entries like this:

```name = Red Hat Satellite Tools 6.7 for RHEL 8 x86_64 (Source RPMs)
baseurl = https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/sat-tools/6.7/source/SRPMS
enabled = 0
gpgcheck = 1
gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
sslverify = 1
sslcacert = /etc/rhsm/ca/redhat-uep.pem
sslclientkey = /etc/pki/entitlement/8454176169347351717-key.pem
sslclientcert = /etc/pki/entitlement/8454176169347351717.pem
metadata_expire = 86400
enabled_metadata = 0
```

Do we need to set "cdn.redhat.com" for NO_PROXY?

Also there is a typo in the Note at step 8, refering to REDH**H**AT_PACKAGE_MIRROR_LOCATION instead of REDHAT_PACKAGE_MIRROR_LOCATION

<img width="976" alt="image" src="https://user-images.githubusercontent.com/23667573/233623703-7472d4e2-4cdb-43f4-b355-476bcf9e5e2b.png">

**Comment by gcoon151** [comment]
@uniquecorn09 I'm guessing you're trying to figure out the mirror location for something not obviously at the bottom of the doc (AWS, Azure, and Google). I highly suspect you're wanting us to document the mirror for softlayer (IBM Cloud). Originally Satellite isn't created for IBM cloud so that's why we didn't put IBM cloud Red Hat mirror specifics. We only included what we did about mirrors as a helper to someone trying to configure the proxy that they'll need to put the details of the mirror in there also.
